<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminMenu</class>
 <widget class="QDialog" name="adminMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>707</width>
    <height>314</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Admin Menu</string>
  </property>
  <widget class="QPushButton" name="buttonCreateProject">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>260</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Create New Project</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonSave">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>260</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Update Project</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listProjects">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>161</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>80</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Project ID:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>200</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Team Size:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="lineDescription">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>140</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineTeam">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>200</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineName">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>110</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineID">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>80</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>110</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Project Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>140</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Desciption:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>My Current Projects</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>453</x>
     <y>20</y>
     <width>20</width>
     <height>281</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>50</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Create New Project</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>20</width>
     <height>281</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="editSize">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>190</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>190</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Team Size:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editName">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>100</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>130</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Desciption:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Project Name:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="editDes">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>130</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>50</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Update/View Project</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonBack">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
